### DISCUSSION TITLE
How to disable the popup after updating LTeX?

### DISCUSSION THREAD

**Discussion by alanlivio** [valentjn/vscode-ltex#71] [Q&A] [answered]
After update LTeX extension, I always see the following po-up info when open VSCode.
How proceed to disable it?
![image](https://user-images.githubusercontent.com/604808/86861364-4fd55180-c09d-11ea-81f9-cc94ea5b4cea.png)

**Comment by valentjn** [comment, marked as answer]
The popup only appears if a language support extension is still installed. You should remove them as the message says. Restart VS Code if that doesn’t help.

See also https://valentjn.github.io/vscode-ltex/docs/transitioning-from-ltex-4x.html and https://valentjn.github.io/vscode-ltex/docs/faq.html#what-happened-to-the-language-support-extensions.

**Comment by alanlivio** [comment]
Thank you @valentjn.
Indeed, I have um ltex-****** extension installed.
